test/core.js

```diff
@@ -347,6 +347,51 @@ describe('Core', () => {
             expect(m.selectors.a).to.shallow.equal(m.mods.a.selectors);
             expect(m.selectors.c).to.shallow.equal(m.mods.c.selectors);
         });
+
+        it('has state-bound selectors from mod branches.', () => {
+
+            const m = MiddleEnd.create({
+                mods: {
+                    a: {
+                        selectors: {
+                            getUpper: ({ a }) => a.toUpperCase()
+                        }
+                    },
+                    b: {},
+                    c: {
+                        selectors: {
+                            multiply: ({ c }, y) => c * y,
+                            some: 'value',
+                            group: {
+                                null: null,
+                                manyArgs: ({ c }, three, four, five) => `${c}${three}${four}${five}`
```

Comment:
the only, extra-nitpicky thing I'd suggest is in documenting this feature, whether in the tests here or in the example composed application in the README, show a selector definition where the first argument is named `state` or something like that.  I got how things worked from reviewing the changes above, just for some reason the call signatures looked odd, I think because the destructured state's included in the definitions here, but not in the calls below; that just took me a sec to put back together and reconcile with what I'd read previously

I'm likely showing my inexperience here, for sure, so might not actually be worth including. Again, very very minor suggestion 😏

Reply:
Is the idea that you want to make it as clear as possible that the first argument to a selector is always the application state?  If that's the case, it's definitely something I take for granted, so I can see how it could lead to confusion if that's not a common understanding.  [This](https://github.com/BigRoomStudios/strange-middle-end/pull/5/files#diff-88333ba11114e014e87198d04680144dR12) might be a good place to note that.

Reply:
🙏 oh man...on thinking about it more, I'm not really sure. By extension, yes? By which I mean, `manyArgs` looked weird to me when called without the state argument first when invoked with `select`, mostly b/c I had to look up how selectors work again and learned there that some segment of app state's always the first argument. Chalking this up to a monday morning sleepy comment / personal learning :trollface: 🎺